styles/components/buttons.css

```diff
@@ -1,11 +1,146 @@
-.button {
+.button,
+a.button {
```

Comment:
dont play with direct element, play by giving class name

Reply:
`button` is the class name

Reply:
here with a.button you are picking a element directly

Reply:
reviewer might confuse between .button class and button element , better you change class name to something like btn

Reply:
fixed - 93e84e4ca91886737163a6787fe428331992a746